gulpfile.js

```diff
@@ -137,4 +139,8 @@ gulp.task('patch', function() {
     runSequence('bump-patch', 'build', 'update-version');
 });
 
-gulp.task('build', ['css', 'js', 'css-min', 'js-min']);
+gulp.task('build', function() {
+    runSequence('lint', 'css', 'js', 'css-min', 'js-min', 'update-version')
```

Comment:
We've talked about that before: why do you use _runSequence_ like this? `css` and `css-min` as well as `js` and `js-min` are independent and can freely run in parallel.